docs/reference/spark-client.md

```diff
@@ -19,17 +19,30 @@ Utilize the power of Spark to interact with the metadata on lakeFS. Possible use
 Please note that Spark 2 is no longer supported with the lakeFS metadata client.
 {: .note }
 
-Start Spark Shell / PySpark with the `--packages` flag:
-
-This client is compiled for Spark 3.1.2 with Hadoop 3.2.1, but can work for other Spark
-versions and higher Hadoop versions.
-
-```bash
-spark-shell --packages io.lakefs:lakefs-spark-client_2.12:0.11.0
-```
-
-Alternatively an assembled jar is available on S3, at
-`s3://treeverse-clients-us-east/lakefs-spark-client/0.11.0/lakefs-spark-client-assembly-0.11.0.jar`
+The Spark metadata client is compiled for Spark 3.1.2 with Hadoop 3.2.1, but
+can work for other Spark versions and higher Hadoop versions.
+
+<div class="tabs">
+  <ul>
+    <li><a href="#spark-shell">`pyspark`, `spark-shell`, `spark-submit`, `spark-sql`</a></li>
```

Comment:
![image](https://github.com/treeverse/lakeFS/assets/9861474/9f260af7-07ac-4bc1-bc47-bdc32c1fbd09)

Not the best looking title, maybe 
`pyspark / spark-shell / spark-submit / spark-sql`

Reply:
Yikes!

Fixed, thanks!